### ISSUE TITLE
make prepare + make build does not build Kitodo

### ISSUE THREAD

**Issue by SvenMarcus** [markusweigelt/kitodo_production_ocrd#43]
When running `make prepare` followed by `make build` the build command fails because `build-resources/kitodo.war` is missing. 
I had to download the Kitodo release files manually to that folder in the Kitodo submodule in order to launch all the services.
This behavior is not documented in the Readme and there should probably just be a build rule to grab the necessary files from the Kitodo release pages if they aren't already there.

**Comment by markusweigelt** [comment]
@SvenMarcus Did you get the fresh state from "master" branch and update the submodule as well?

**Comment by SvenMarcus** [comment]
Yes, I actually completely removed the previous project and cloned fresh with submodules

**Comment by markusweigelt** [comment]
Can you add `--no-cache` to the `docker-compose build` in `make build` target. Which Docker Compose Version do you use? Cause there can be some problems regarding variable substitution https://github.com/markusweigelt/kitodo_production_ocrd/issues/39#issuecomment-1237033989_

**Comment by SvenMarcus** [comment]
We are on `docker-compose` version 1.29.2, which should be the latest version before it became a plugin of Docker itself.
I recloned everything, deleted all the images and tried with `--no-cache`, but Kitodo still doesn't build

**Comment by markusweigelt** [comment]
Yes in my case i run the the `Docker Compose` v2.10.2 with overwritten alias `docker-compose` cause it is called `docker compose`. We should clarify next week on the conference call which version we are working with, if there is any major difference.

I tested this too with new clone of repository and submodule checkout and it works. Are there some config differences to repository for example `.env` adjustments etc. What branch do use?
